### DISCUSSION TITLE
What does it mean "available only for managed applications"?

### DISCUSSION THREAD

**Discussion by djeikyb** [AvaloniaUI/Avalonia.Xaml.Behaviors#163] [Q&A] [answered]
The readme says:

> Avalonia port is available only for managed applications.

What is "managed application" shorthand for?

**Comment by maxkatz6** [comment, marked as answer]
.NET managed code - is a code based on Common Language Runtime, used for C#, F#, VB and more.
https://learn.microsoft.com/en-us/dotnet/standard/managed-code

It just means, that you can't use this library (as well as Avalonia itself), with C++ native apps (or, "unmanaged"), for example.
